core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -983,11 +984,19 @@ private[spark] class SparkSubmit extends Logging {
         e
     }
 
+    var DriverPodIsNormal: Boolean = if (args.master.startsWith("k8s")) true else false
+    var driverThrow: Throwable = null
     try {
       app.start(childArgs.toArray, sparkConf)
     } catch {
       case t: Throwable =>
-        throw findCause(t)
+        logWarning("Some ERR/Exception happened when app is running.")
+        if (args.master.startsWith("k8s")) {
```

Comment:
Did you want to use `DriverPodIsNormal` here?

Reply:
Yes, `DriverPodIsNormal` applied here may be better.

Reply:
Then, please use it.